Src/Base/AMReX_REAL.H

```diff
@@ -66,9 +66,28 @@ typedef double amrex_particle_real;
 
 #ifdef __cplusplus
 namespace amrex {
+    /** Floating Point Type for Fields
+     *
+     * This is a floating point type used by default for fields such as
+     * FabArray and MultiFab classes. This can be changed at compile time to
```

Comment:
FabArray does not have a default type. Maybe change it to FArrayBox. Also, what's the experimental floating point precision you mention below?

Reply:
Thanks, typo fixed!

I meant to write: this is the location where you would play with FP16 :)